README.md

```diff
@@ -36,29 +36,39 @@ This project is still in its very early days. Expect some bugs.
 ### Run the latest release
 
 ```bash
-git clone https://github.com/nashsu/FreeAskInternet.git
+git clone https://github.com/chris-ss/FreeAskInternet.git
```

Comment:
This PR is changing the readme URLs of the repo to the fork. FYI @ilkocn 

Also changes the docker container away from nashsu account to duriancat.

Seems like the changes are unrelated to the PR so just making sure...

Reply:
Correct